opencsp/common/lib/camera/ImageAcquisitionAbstract.py

```diff
@@ -84,62 +87,62 @@ def _check_saturated(im):
         if not _check_exposure_set():
             raise ValueError('Error with setting exposure.')
 
-        # Set final exposure and print results
+        # Set final exposure and log results
         exposure_value_set = exposure_values[_get_exposure_idxs()[0]]
         self.exposure_time = exposure_value_set
-        print(f'Exposure set to: {exposure_value_set}')
+        lt.info(f'Exposure set to: {exposure_value_set}')
 
     @abstractmethod
     def get_frame(self):
         """Gets a single frame from the camera"""
-        pass
 
-    @abstractproperty
+    @property
+    @abstractmethod
     def gain(self):
         """Camera gain value"""
-        pass
 
-    @abstractproperty
+    @property
+    @abstractmethod
     def exposure_time(self) -> int:
         """Camera exposure_time value (microseconds)"""
-        pass
 
     @property
     def exposure_time_seconds(self) -> float:
+        """Camera exposure_time value (seconds)"""
         return self.exposure_time / 1_000_000
 
     @exposure_time_seconds.setter
     def exposure_time_seconds(self, exposure_time_seconds: float):
-        self.exposure_time = int(exposure_time_seconds / 1_000_000)
+        """Camera exposure_time value (seconds)"""
+        self.exposure_time = int(exposure_time_seconds * 1_000_000)

```

Comment:
@bbean, I believe this was a bug, but only found it when I was reviewing my code here. Let me know if you agree.

Reply:
It totally was a bug. Thank you!